src/components/StreamerConnection.vue

```diff
@@ -143,6 +143,11 @@ export default {
         } catch (e) {}
       };
 
+      this.connection.onUserIdAlreadyTaken = (useridAlreadyTaken) => {
```

Comment:
So I am noticing that when I test this it _does_ start a stream. 
I'm not sure if we can check before the share screen pops up? If not, we should immediately end the stream.

Reply:
This is probably outside the scope of my abilities and should probably be a new issue. It doesn't start a new room, so the tab is streaming into the unknown.